ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml

```diff
@@ -43,18 +32,51 @@ ScrollView {
         Layout.fillWidth: true
         Layout.fillHeight: true
 
+        Timer {
+            id: timer
+        }
+
+        function scrollToBottom(force, caller) {
+            if (!force && !chatLogView.atYEnd) {
+                // User has scrolled up, we don't want to scroll back
+                return
+            }
+            if (caller) {
+                if (caller !== chatLogView.itemAtIndex(chatLogView.count - 1)) {
+                    // If we have a caller, only accept its request if it's the last message
+                    return
+                }
+                // Add a small delay because images, even though they say they say they are loaed, they aren't shown yet
+                timer.setTimeout(function() {
+                    Qt.callLater(chatLogView.positionViewAtEnd)
```

Comment:
Have you tried this with a really big image? Just curious if the `Image.Status` property waits for downloading to occur, and this additional `100ms` delay is simply there for the UI to update with the downloaded image.

Reply:
Just tested and it works